app/views/users/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<div id="show-user-profile">
+      <img src="https://img.icons8.com/external-sbts2018-outline-sbts2018/100/000000/external-profile-seo-4.-1-sbts2018-outline-sbts2018.png"/>
+
+      <p class="user-email"><%= @user.email %></p>
+
+      <% unless @actual_current_user %>
+        <%= link_to "Follow", user_relationships_create_path(@user), method: :post, class: 'btn btn-success' unless @rel.present? %>
```

Comment:
The @rel instance variable is not necessary, we could work the relationship type with the user controller, for example in this case we could do something like this: `user.followers.include? current_user`

Reply:
All changes were applied in the following commits of this same PR